ruby/spec/asciidoctor_kroki_block_macro_spec.rb

```diff
@@ -92,7 +104,7 @@ def resolve_target_path(target)
     it 'should display unresolved block macro message when the target cannot be resolved' do
       # noinspection RubyClassModuleNamingConvention
       class UnresolvedTargetKrokiBlockMacroProcessor < ::AsciidoctorExtensions::KrokiBlockMacroProcessor
-        def resolve_target_path(_target)
+        def resolve_target_path(_target, _parent)
```

Comment:
We are not using the parameters but I guess it should be `_parent` (first parameter) then `_target`
```suggestion
        def resolve_target_path(_parent, _target)
```

Reply:
Will fix